examples/karaf-rest-example/karaf-rest-example-provider/src/main/java/org/apache/karaf/examples/rest/BookingService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.karaf.examples.rest;
+
+import java.util.List;
+
+/**
+ * Simple interface describing the booking service.
+ */
+public interface BookingService {
```

Comment:
To show how to decouple REST from service, maybe this could have been isolated in a dedicated module. Thoughts ?

Reply:
We could have :
* **api** module containing model and service interface
* **provider** module containing REST impl :
  * *Apache CXF*
  * *Apache Aries* JAXRS whiteboard
* **client** module containing REST client :
  * *Java HTTP client*
  * *Apache CXF*